runtime/aardvarks/info.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apigeeregistry/v1
+items:
+  - apiVersion: apigeeregistry/v1
+    kind: API
+    metadata:
+      name: apigee-apihub-demo-fauna-aardvarks-proxy
+      labels:
+        apihub-business-unit: apigee-apihub-demo
+        apihub-kind: proxy
+        source: generate-animal-apis
```

Comment:
We're generally using prefixed names for labels and annotations... I wonder if that should be the common pattern? Otherwise, should we define rules for what should / shouldn't be prefixed?

Reply:
right, so maybe `apihub-source`... or OTOH, I think that `apihub-kind` might be a more general `kind`... and all this needs documentation

Reply:
Yes, assuming that signals are a exclusively a feature of API Hub, I think we'll likely want this to be `apihub-source` so that sources are tracked in an API Hub taxonomy.

Reply:
This is `apihub-source` now.